x/nexus/client/cli/query.go

```diff
@@ -90,3 +92,35 @@ func GetCommandLatestDepositAddress() *cobra.Command {
 
 	return cmd
 }
+
+// GetCommandTransfersForChain returns the query for the transfers for a given chain
+func GetCommandTransfersForChain() *cobra.Command {
+	cmd := &cobra.Command{
```

Comment:
our transfers can be in 3 states. 
[confirmed, but pre create-pending-transfers]. 
[confirmed & created in a pending-transfer batch, but pre-signing]. 
[confirmed & batched created & signed]. 
is this intended to return all 3 types of transfers / states?

Reply:
we should rename the commands in the bridge modules instead. This is correct as `pending`